src/index.js

```diff
@@ -18,22 +18,39 @@ function _panic(msg, data) {
     return new Error(data || msg);
 }
 
-function _isRunAsCustomization(components) {
-    return components[1] === "workspaces" && components[3] === "customization";
+function _getUrlComponents() {
+    let path = window.location.pathname;
+    let components = path.split('/');
+
+    let workspaceIndex = components.lastIndexOf('workspaces');
+
+    if (workspaceIndex == -1) {
+        return {
+            basePath: '',
+            customizedWorkspaceId: null
+        }
+    }
+
+    let customizedWorkspaceId = components[workspaceIndex + 3] === "customization"
```

Comment:
Should it not be `+2`? If the URL is /workspaces/:workspaceID/customization then woskspace ID is +1 and "customization" is +2.